setup.py

```diff
@@ -21,7 +21,7 @@ def readme():
                       'SQLAlchemy>=1.2.2',
                       'Flask-SQLAlchemy>=2.3',
                       'packaging>=14.1'],
-    extras_require={'tests': ['pytest-postgresql<2.0', 'psycopg2-binary']},
+    extras_require={'tests': ['pytest-postgresql>=2.4.0', 'psycopg2-binary']},
```

Comment:
Strictly speaking it may have been possible to retain Python 3.5 support by specifying `pytest-postgresql>=2.3.0` since [that version supports py3.5](https://github.com/ClearcodeHQ/pytest-postgresql/blob/5ef7c74b2cb821bc8f7a4af125675d8eb15e44da/setup.py#L72) and the additional credential arguments, but I figured based on some of the version bumps since then that it might be better to roll forward.